src/shared/utils/config-utils/fetch-event-names.test.ts

```diff
@@ -0,0 +1,21 @@
+import { fetchEventNames } from "./fetch-event-names";
+
+jest.mock("./fetch-vendor-service-config", () => ({
+  fetchVendorServiceConfig: () => [
+    { event_name: "BOUGHT_EGGS" },
+    { event_name: "MADE_CAKE" },
+    { event_name: "ATE_CAKE" },
+    { event_name: "MADE_CAKE" },
+    { event_name: "MADE_TEA" },
+    { event_name: "ATE_CAKE" },
```

Comment:
Is there a failure case with 'HAVE_CAKE' and 'EAT_CAKE' somewhere? ;-)

Reply:
Hmm I tried it but somehow the test still passes 🤔